var/spack/repos/builtin/packages/nlohmann-json/package.py

```diff
@@ -38,12 +40,11 @@ class NlohmannJson(CMakePackage):
 
     def cmake_args(self):
         spec = self.spec
+        define = CMakePackage.define
```

Comment:
since this is a derived `class NlohmannJson(CMakePackage)`, don't we have access to `define()` already?

Reply:
Yeah,  but I do it for the same reason as you setting `self = self.spec` and using `spec` directly -- I think it  reads a little bit clearer. I can delete either or both if you have a preference.

Reply:
I see, so would `self.define` work, too?